lib/events.js

```diff
@@ -36,6 +36,7 @@ EventEmitter.usingDomains = false;
 
 EventEmitter.prototype.domain = undefined;
 EventEmitter.prototype._events = undefined;
+EventEmitter.prototype._eventsCount = 0;
```

Comment:
Why is this necessary? It seems to me that it used to be necessary for `process` in `bootstrap_node.js` since C++ pre-sets `_events`, which prevents the `EventEmitter` constructor from setting `_eventsCount`. But now that you've removed the C++ part of this, merely removing `_eventsCount` in `bootstrap_node.js` should suffice. I'm probably mistaken though.

Reply:
Can we use a private `Symbol` if it is necessary?

Reply:
The issue is that this has been public for a while anyway. I'm just adding it onto the prototype since there are packages out there that use EventEmitter without using `EventEmitter.prototype.init` which means that they end up changing the hidden class when they first call `emit` or `addListener`.

The reason it wasn't there before — at least as far as I understand — is that there were issues with performance around it.

Reply:
Oh ok then.